### PR TITLE
Get rid of links to the gEDA launchpad bug tracker.

### PR DESCRIPTION
**Pull request by vzh**
All links have been replaced with link to lepton issue tracker.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -247,12 +247,10 @@ There are several ways to get help with installing and using gEDA:
...
 
- - Alternatively, you can add your question to the gEDA Answers page
-   on Launchpad. <https://answers.launchpad.net/geda>
-
  - If you have discovered a bug, have a feature request, or have
-   written a patch to gEDA, please create an item on the gEDA Bugs
-   page on Launchpad. <http://bugs.launchpad.net/geda>
+   written a patch to lepton-eda, please create an item on the
+   lepton-eda bug tracker page:
+   <https://github.com/lepton-eda/lepton-eda/issues>
```
Comment (peter-b) [inline README]: Maybe this should refer to making a pull request?

**Comment by vzh** [comment]
@peter-b I'm novice here, you know, and I just changed addresses, didn't added other info. If you add some info about PR's in README I'm all for it! I think we should mention both ways.

File: README (hunk 1 above)
Comment (gareth8118) [inline README]: PRs are the Github Way and allow evolution and commentary that's more difficult to correlate than attached patches - I definitely think they should be encouraged. don't want to rule out an old school .patch bundle though.

**Comment by peter-b** [comment]
@vzh I'll sort it out when the new guide for contributors (in #42) is merged... I think it makes  sense for the readme file to say, "go and look at the contributors' guide"

**Comment by peter-b** [comment]
@vzh To get the CI build to pass, just rebase this branch on `master` and that should fix it, now that #40 has been merged in.

**Pull request merged**